bots/issue-scan

```diff
@@ -165,11 +181,21 @@ def output_task(command, issue, verbose):
             context = pipes.quote(context)
         return (checkout + cmd).format(
             issue=int(number),
-            priority=PRIORITY,
+            priority=distributed_queue.MAX_PRIORITY,
             name=name,
             context=context,
         )
 
+def queue_task(channel, result):
+    body = {
+        "command": result,
+        "type": "issue",
+    }
+    if contains_any(result, REDHAT_TASKS):
+        channel.basic_publish('', 'rhel', json.dumps(body), properties=pika.BasicProperties(priority=distributed_queue.MAX_PRIORITY))
+    else:
+        channel.basic_publish('', 'public', json.dumps(body), properties=pika.BasicProperties(priority=distributed_queue.MAX_PRIORITY))
```

Comment:
Please factor out the "map a task name to a queue name" part (this could go into tasks as well, supposedly we use this code several times?), and do the actual basic_publish() call just once.

Reply:
tests-scan uses this bit as well; i'll move it out